app/services/account_reset/cancel_request_for_user.rb

```diff
@@ -1,22 +0,0 @@
-# frozen_string_literal: true
-
-module AccountReset
-  class CancelRequestForUser
-    attr_reader :user
-
-    def initialize(user)
-      @user = user
-    end
-
-    def call(now: Time.zone.now)
-      account_reset_request.update!(cancelled_at: now)
-      NotifyUserOfRequestCancellation.new(user).call
```

Comment:
Since this was only used here, perhaps one way to reduce the amount of files you need to read to understand this is to pull in the notification into this class.